assets/src/blocks/Articles/Articles.js

```diff
@@ -11,12 +11,36 @@ import TagSelector from '../../components/TagSelector/TagSelector';
 import PostSelector from '../../components/PostSelector/PostSelector';
 import PostTypeSelector from '../../components/PostTypeSelector/PostTypeSelector';
 import {URLInput} from "../../components/URLInput/URLInput";
-
+import { v4 as uuidv4 } from 'uuid';
 
 const TextControl = withCharacterCounter( BaseTextControl );
 const TextareaControl = withCharacterCounter( BaseTextareaControl );
 
 export class Articles extends Component {
+  constructor(props) {
+    super(props);
+    this.instanceName = `planet4-blocks-articles-${ uuidv4() }`;
+  }
+
+  componentDidMount() {
+    this.lazyLoadInterval = setInterval(() => {
+      if (document.querySelector(`.${this.instanceName} section.articles-block`)) {
+        this.updateLazyLoad();
+        clearInterval(this.lazyLoadInterval);
+      }
+    }, 250);
```

Comment:
Too bad `ServerSideRender` doesn't provide a callback when rendering is complete, though I understand why they don't provide it as it's probably not a good idea in most cases, except in this one it would probably be somewhat better than polling. But this indeed looks like the only way to achieve what we want :+1:

Reply:
Another option could be to extend the `ServerSideRender` component, as it's pretty easy to implement the `onChange` handler, like here https://github.com/WordPress/gutenberg/issues/8379#issuecomment-446815846 . That would make it require a lot less code to apply the lazy loading update to multiple blocks. But maybe it won't work as it's only called when the `ServerSideRender` component updates, which is just before it renders.

Reply:
@lithrel  @Inwerpsel  I did a few tests to wrap it but couldn't get to a decent solution, and since we are rolling out the ServerSideRender in the near future, and this block will be converted to WYSIWYG, this looks like a reasonable tradeoff for now. 

Maybe a more elegant option for the WYSIWYG blocks would be to use one of the lazy-load React components out there like these ones:
https://www.npmjs.com/package/react-lazyload
https://github.com/Aljullu/react-lazy-load-image-component

I guess we can go ahead with this fix in the meantime and create a ticket to revisit the lazy-load implementation towards a "Reactier" (?) approach for the WYSIWYG blocks.